soundata/datasets/dcase_bioacoustic.py

```diff
@@ -0,0 +1,550 @@
+        super().__init__(clip_id, data_home, dataset_name, index, metadata)
+
+        self.audio_path = self.get_path("audio")
+        self.csv_path = self.get_path("csv")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The clip's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    @property
+    def split(self):
+        """The data splits (e.g. train)
+
+        Returns
+            * str - split
+
+        """
+        return self._clip_metadata.get("split")
+
+    @property
+    def subdataset(self):
+        """The (sub)dataset
+
+        Returns
+            * str - subdataset
+
+        """
+        return self._clip_metadata.get("subdataset")
+
+    @core.cached_property
+    def events_classes(self) -> Optional[list]:
+        """The audio events
+
+        Returns
+            * list - list of the annotated events
+
+        """
+        return load_events_classes(self.csv_path)
+
+    @core.cached_property
+    def events(self) -> Optional[annotations.Events]:
+        """The audio events
+
+        Returns
+            * annotations.Events - audio event object
+
+        """
+        return load_events(self.csv_path)
+
+    @core.cached_property
+    def POSevents(self) -> Optional[annotations.Events]:
+        """The audio events for POS (positive) class
+
+        Returns
+            * annotations.Events - audio event object
+
+        """
+        return load_POSevents(self.csv_path)
+
+    def to_jams(self):
+        """Get the clip's data in jams format
+
+        Returns:
+            jams.JAMS: the clip's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            audio_path=self.audio_path,
+            events=self.events,
+            metadata={
+                "split": self._clip_metadata.get("split"),
+                "subdataset": self._clip_metadata.get("subdataset"),
+            },
+        )
+
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO, sr=None) -> Tuple[np.ndarray, float]:
+    """Load a DCASE bioacoustic audio file.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to audio file
+        sr (int or None): sample rate for loaded audio, None by default, which
+            uses the file's original sample rate without resampling.
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+
+    """
+    audio, sr = librosa.load(fhandle, sr=sr, mono=True)
+    return audio, sr
+
+
+@io.coerce_to_string_io
+def load_events(fhandle: TextIO) -> annotations.Events:
+    """Load an DCASE bioacoustic sound events annotation file
+
+    Args:
+        fhandle (str or file-like): File-like object or path to the sound events annotation file
+        positive (bool): False get all labels, True get just POS labels
```

Comment:
I think this `positive` argument passed in the function, am I right? Maybe remove the line in the docs? Since it is not used in the function